### PR TITLE
fix(github): update issue templates

### PR DESCRIPTION
**Pull request by raisedadead**
I noticed that most users report in wrong categories including seasoned @freeCodeCamp/moderators - likely due to the fact the usually one would be in a rush to just report a bug. This update should help a bit.

### CODE REVIEW

**Review by ahmaxed — DISMISSED** [review]

File: .github/ISSUE_TEMPLATE/02--issues-with-software-on-platforms.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-name: Report issues with software on freeCodeCamp.org's platforms
+name: Issue - User Interface (UX) on our Platforms
```
Comment (ojeytonwilliams) [inline .github/ISSUE_TEMPLATE/02--issues-with-software-on-platforms.yml:1]:
```suggestion
name: Issue - User Experience (UX) on our Platforms
```
:eyes:

Comment (raisedadead) [reply .github/ISSUE_TEMPLATE/02--issues-with-software-on-platforms.yml:1]: TL;DR: This is intentional

I wanted to use all the common terms UI, UX, and the full form of the most common of both User Interface, all in a concise and tiny title.

The goal is people should not need to read or think.

Comment (SethFalco) [reply .github/ISSUE_TEMPLATE/02--issues-with-software-on-platforms.yml:1]: To be honest, I think this creates a bit of confusion. 🤔 
When a term is followed by an acronym in brackets, readers associate the two.

Someone who isn't already familiar with "UX" will think "UX" is short for User Interface. (Which isn't unlikely given the target audience of freeCodeCamp.)

People that know what "UX" means, will likely open a PR or issue to suggest changing it. (Such as this suggestion that was made just now.)

Comment (raisedadead) [reply .github/ISSUE_TEMPLATE/02--issues-with-software-on-platforms.yml:1]: Yes - Once in a blue moon, people may try to change it as a nit-pick. You correctly identified that. Maybe dropping the acronym is the best.

**Comment by SethFalco** [comment]
Might be off-topic to ask this in this PR, but just a question!

I actually had no idea we were supposed to open an issue here when we see a typo or dead link on articles. ^-^'
I always emailed in, and it'd be promptly fixed.

In future, should I be favoring opening an issue instead? (Or alternatively, should emailing the editorial team directly be added as an option on that template?)

**Review by naomi-lgbt — APPROVED** [review]

**Review by ahmaxed — APPROVED** [review]

**Pull request merged**